catalog/controller/extension/payment/cardinity.php

```diff
@@ -41,6 +41,16 @@ public function index()
 				$data['order_id'] = $formattedOrderId; //$this->session->data['order_id'];
 				$data['description'] = $this->session->getId(); //  'OC' . $this->session->data['order_id'];
 				$data['return_url'] = $this->url->link('extension/payment/cardinity/externalPaymentCallback', '', true);
+
+                if (!empty($order_info['email'])){
+                    $data['email_address'] =  $order_info['email'];
+                }
+                if (!empty($order_info['telephone'])){
+                    $data['mobile_phone_number'] =  $order_info['telephone'];
+                }
+
+                $this->testLog("Requesting".print_r($data, true));
```

Comment:
Is it necessary to keep the logging? Won't it log any sensitive data?

Reply:
its external request, so no sensitive data. Removed anyways.